naughty/rhel-8/1453-fsadm-unbound-variable

```diff
@@ -0,0 +1 @@
+/usr/sbin/fsadm: line *: $3: unbound variable
```

Comment:
`warning: /sbin/fsadm: line 815: $3: unbound variable`
just the path is wrong, not `/usr/bin/` but `/sbin/`

Reply:
Hah, thanks for spotting! Pushed.

Reply:
Yay, worked now!